### DISCUSSION TITLE
@typescript-eslint/typedef doesn't support allowExpressions option

### DISCUSSION THREAD

**Discussion by antonioOrtiz** [eslint/eslint#17394] [Q&A] [answered]
```
`package.json`
{
    "devDependencies": 
    ....other dependencies
    "@typescript-eslint/eslint-plugin": "^6.0.0",
    "eslint": "^8.45.0",
    ....other dependencies
}
```

```
.eslintrc.js
    '@typescript-eslint/typedef': [
    'error',
    {
    allowExpressions: true,
    variableDeclaration: true,
    arrowParameter: false
    }
],
```

vscode is alerting me as well as when I make a commit:
![2023-07-20 11_08_20-package json - documentengine-service (Workspace) - Visual Studio Code](https://github.com/eslint/eslint/assets/4282013/bcfa9847-ce9c-4772-9859-1aab654f31c4)

What should about this rule with my current dependencies? Is there a workaround?

**Comment by nzakas** [comment, marked as answer]
This is actually an issue with [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint) rather than ESLint itself. Please open an issue there.

**Reply by antonioOrtiz** [reply]
Thank you!
